### PR TITLE
Rearrange Footer, move around Social Icons

### PR DESCRIPTION
**Pull request by lauragift21**
This implements the changes requested by @jlooper.
- Move Social Icons below the Patreon and Workshop buttons
- Add Contact Us and Press Kit to Footer
- Add GitHub Icon to Navbar

zulynksxv7

### CODE REVIEW

**Comment by NataliaTepluhina** [comment]
@lauragift21 thank you for your work! Some comments:
1. PRs should be created using `dev` as a base branch, please check the instruction posted in Slack chapter leaders channel. I will change base for this PR.
2. Tbh I don't like Jen's idea of placing footer icons under CTA buttons, it doesn't look good
@jlooper could you please reconsider this from design point of view?

File: components/Header.vue [hunk 1]
```diff
@@ -13,6 +13,9 @@
       <v-btn flat to="/library" nuxt>{{ $t("menu.library") }}</v-btn>
       <v-btn flat href="https://dev.to/vuevixens" target="_blank">Blog</v-btn>
       <v-btn flat href="https://vuevixens.threadless.com/" target="_blank">{{ $t("menu.shop") }}</v-btn>
+      <v-btn flat href="https://github.com/VueVixens" target="_blank">
+        <img src="~/assets/images/github.png" alt="GitHub Logo">
```
Comment (NataliaTepluhina) [inline components/Header.vue]: It's better to use the same icon we used for GitHub link in footer, i.e. `<i class="fab fa-github">`

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,10635 @@
+{
```
Comment (NataliaTepluhina) [inline package-lock.json]: Please don't commit lock files to the repository

**Comment by lauragift21** [comment]
@NataliaTepluhina Thanks for the review. I'll fix the issues you pointed out while we wait for @jlooper to review the icons under the cta buttons.

**Comment by jlooper** [comment]
Yes, I agree that the social icons turned out to be pretty awful, under those buttons. Apologies for requesting that! 

Let's do this:

- put the newsletter, twitter, and slack icons back to the footer but leave GitHub's icon in the header.

- remove "A project ... Progress" and "Request a Skulk" buttons entirely from the footer.

The rest was looking good. Thank you for this contribution!

**Comment by lauragift21** [comment]
Alright I'll make those adjustments.

**Pull request merged**